package.json

```diff
@@ -50,7 +50,7 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "test:ci": "react-scripts test --coverage --testResultsProcessor ./node_modules/jest-sonar-reporter",
+    "test:ci": "react-scripts test --coverage --testResultsProcessor ./node_modules/jest-sonar-reporter -- -u",
```

Comment:
What's this?

Reply:
Snapshot tests keep failing due to floating point differences. Was hoping that a forced update would fix them.